### PR TITLE
Release change

### PR DESCRIPTION
**Pull request by langner**
This does the routine stuff:
* bumps the version in various files
* updates the changelog
* updated the THANKS file

I've also taken the liberty to restructure the THANKS file a bit, hopefully it's more useful.

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,27 @@
+Changes in cclib-1.6.1
+
+Features:
+	* New attribute nsocoeffs for natural spin orbital coefficients (Shiv Upadhyay)
+	* New attribute nsooccnos for natural spin orbital occupation numbers (Shiv Upadhyay)
+    * New methods: alpha and beta electron counts (Jaime Rodríguez-Guerra)
+	* Support coreelectrons attribute in Molcas (Kunal Sharma)
+	* Support etoscs for response calculations in Falton (Peter Reinholdt)
```
Comment (berquist) [inline CHANGELOG]:
```suggestion
	* Support etoscs for response calculations in Dalton (Peter Reinholdt)
```

File: ANNOUNCE [hunk 2]
```diff
@@ -36,8 +36,9 @@ The cclib development team
...
-    * Support grads attribute in Psi4 and Molpro
-    * Support optstatus for IRCs and in Psi4
-    * Updated test file versions to Gaussian16
-    * Many other minor improvements and bug fixes
+* New attributes nsocoeffs and nsooccnos for natural spin orbital coefficients (Shiv Upadhyay)
+* New methods for alpha and beta electron counts (Jaime Rodríguez-Guerra)
+* Support coreelectrons attribute in Molcas (Kunal Sharma)
+* Support etoscs for response calculations in Dalton (Peter Reinholdt)
+* Updated testing framework (Jaime Rodríguez-Guerra, Maxim Stolyarchuk and others)
+* Many other minor improvements and bug fixes from Alessandro Genova. Felipe S. S. Schneider and others
```
Comment (ATenderholt) [inline ANNOUNCE]: Comma after Genova?

Comment (langner) [reply ANNOUNCE]: Done.

**Comment by ATenderholt** [comment]
I'll leave it to you to merge when you're ready.

**Pull request merged**